doc/source/explanation-differential-privacy.rst

```diff
@@ -1,3 +1,10 @@
+:og:description: Learn how differential privacy ensures data security in federated learning, with central and local approaches to prevent leakage of sensitive information during model updates and training.
```

Comment:
I know that this explainer is not Flower specific, but should we still mention Flower in the description for SEO?